src/offerings.ts

```diff
@@ -0,0 +1,343 @@
+
+export enum PACKAGE_TYPE {
```

Comment:
wasn't sure how to name this file tbh. It doesn't just have offerings, but rather all of the types related to offerings. I didn't want to split it into 20 files, it felt like a bit much.